ouroboros-consensus-cardano/src/shelley/Ouroboros/Consensus/Shelley/Protocol/Praos.hs

```diff
@@ -59,6 +59,11 @@ data PraosEnvelopeError
     -- for this era this specific node's configuration (the second 'Version').
     -- The only thing the header did "wrong" was extend such a ledger state.
     --
+    -- FIXME: It's not clear to me what the second component represents here.
```

Comment:
I misinterpreted the second component of `ObsoleteNode` as a minor version. We can remove this `FIXME` (no need to clarify anything).